### PR TITLE
PHPUnit: upgrade to 5.4+

### PR DESCRIPTION
**Pull request by keradus**
With this PR, we are not using anything deprecated on PHPUnit 5.4 and lower

sadly, cannot go to 2.2 line as PHPUnit 5+ requires PHP 5.6+

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Milestone is incorrect still.

**Comment by keradus** [comment]
please read carrefully

 >Milestone v2.2.5

> keradus changed the base branch to FriendsOfPHP:2.3 from FriendsOfPHP:2.2

> sadly, cannot go to 2.2 line as PHPUnit 5+ requires PHP 5.6+

sorry, but I'm not sure what value your comment provides ...

File: composer.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "require-dev": {
         "johnkary/phpunit-speedtrap": "^1.0.1",
         "justinrainbow/json-schema": "^5.0",
-        "phpunit/phpunit": "^4.8.35 || ^5.4.3",
+        "phpunit/phpunit": "^5.4.3",
```
Comment (keradus) [inline composer.json]: we need to update schema ref in phpunit xml file

**Comment by GrahamCampbell** [comment]
> sorry, but I'm not sure what value your comment provides ...

I as pointing out the milestone is incorrect, and has now been fixed:

image

**Comment by TomasVotruba** [comment]
:clap: great job!

File: composer.json (hunk 1 above)
Comment (localheinz) [reply composer.json]: Hehe, not anymore, see #3013!

**Comment by keradus** [comment]
replaced by #3182

**Pull request closed without merging**